test/Microsoft.DotNet.Tools.Tests.Utilities/TestBase.cs

```diff
@@ -53,21 +39,16 @@ protected static TestAssets TestAssets
                 {
                     var assetsRoot = Path.Combine(RepoRoot, "TestAssets");
 
-                    s_testAssets = new TestAssets(new DirectoryInfo(assetsRoot)); 
+                    s_testAssets = new TestAssets(
+                        new DirectoryInfo(assetsRoot),
+                        new FileInfo(new EnvironmentProvider().GetCommandPath("dotnet")),
```

Comment:
This was intentional, though we should probably chat about it. We originally had an intent to be able to test the dotnet on the PATH but it seems to only have caused us grief. Do we want the change?